src/script/generic.ts

```diff
@@ -95,7 +95,7 @@ export function extractGeneric(element: VElement): GenericProcessInfo | null {
         typeDefScope: Scope,
         isRemoveTarget: (nodeOrToken: HasLocation) => boolean,
     ) {
-        for (const variable of [...typeDefScope.variables]) {
+        for (const variable of typeDefScope.variables) {
```

Comment:
It's just sharing, we had to copy the original array because it might be modified, so we'll undo this modification.

Reply:
The tests should have caught that, but it seems I broke them. Sorry 😅